pontoon/contributors/static/js/profile.js

```diff
@@ -107,39 +107,35 @@ var Pontoon = (function (my) {
               },
             },
             scales: {
-              xAxes: [
-                {
-                  type: 'time',
-                  time: {
-                    displayFormats: {
-                      month: 'MMM',
-                    },
-                    tooltipFormat: 'MMMM YYYY',
-                  },
-                  gridLines: {
-                    display: false,
-                  },
-                  offset: true,
-                  ticks: {
-                    source: 'data',
+              x: {
+                type: 'time',
+                time: {
+                  displayFormats: {
+                    month: 'MMM',
                   },
+                  tooltipFormat: 'MMMM YYYY',
                 },
-              ],
-              yAxes: [
-                {
-                  gridLines: {
-                    display: false,
-                  },
-                  position: 'right',
-                  ticks: {
-                    beginAtZero: true,
-                    maxTicksLimit: 3,
-                    max: 100,
-                    precision: 0,
-                    callback: (value) => nf.format(value / 100),
-                  },
+                grid: {
+                  display: false,
+                },
+                offset: true,
+                ticks: {
+                  source: 'data',
                 },
-              ],
+              },
+              y: {
+                grid: {
+                  display: false,
+                },
+                ticks: {
+                  beginAtZero: true,
+                  maxTicksLimit: 3,
+                  precision: 2,
+                  callback: (value) => nf.format(value),
```

Comment:
Why the change in the `precision` and the `callback` settings?

There are a few things that don't work as expected in the new Profile chart:
1. The curve doesn't have a background.
2. The Y scale data is off.
3. The tooltip: items overlap, the data format is off and there is no border.
4. The chart is cut-off at the top.
5. Can we keep the line curved?

<img width="478" alt="before" src="https://github.com/mozilla/pontoon/assets/626716/21dd1787-80f3-46d6-9814-783c0124dc4a">
<img width="477" alt="after" src="https://github.com/mozilla/pontoon/assets/626716/ece50dbc-5dd3-41e5-9cfc-7c30e1ac1ed5">